### PR TITLE
[DOCS-6469] Docker image updates for 6.2,7.0,7.1

### PR DESCRIPTION
**Pull request by gravitonian**

### CODE REVIEW

File: content-services/6.2/install/containers/index.md [hunk 1]
```diff
@@ -49,30 +49,25 @@ It's important to understand the difference between using containers and using V
 The main difference is that when you run a container, you are not starting a complete new OS instance. This makes containers much more lightweight and quicker to start. A container also takes up much less space on your hard-disk as it doesn't have to ship the whole OS.
 
 ## Alfresco Docker images
+The public Alfresco Docker images are available in the [Docker Hub](https://hub.docker.com) registry. There are also
+private Enterprise-only images in the Quay.io registry. Access to these can be requested from
+[Hyland Community](https://community.hyland.com/)
```
Comment (anxumalo) [inline content-services/6.2/install/containers/index.md:54]: Multiple occurrences:
```suggestion
[Hyland Community](https://community.hyland.com/){:target="_blank"}.
```

**Review by anxumalo — COMMENTED** [review]
Looks good - just a few suggested updates inline

**Review by anxumalo — APPROVED** [review]
@gravitonian See inline comments for further feedback

File: content-services/6.2/install/containers/customize.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ You now need to install the AMP files into the Alfresco Content Repository image
 
 5. Replace the image used by the **alfresco** service in the Docker Compose file you chose in step 1.
 
-    For example, replace `image: alfresco/alfresco-content-repository:6.2.2.1` with `image: myregistrydomain/my-custom-alfresco-content-repository:6.2.2.1`:
+    For example, replace `image: quay.io/alfresco/alfresco-content-repository:6.2.2` with `image: myregistrydomain/my-custom-alfresco-content-repository:6.2.2.1`:
```
[the remark this answers is not in the document]
Comment (gravitonian) [reply content-services/6.2/install/containers/customize.md:85]: No, the build instruction uses that version number.

**Review by anxumalo — APPROVED** [review]

**Pull request merged**